mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent {
+        docker {
+            image 'rocm/mlir:miopen_test'
+            args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+            label 'gfx908'
+        }
+    }
+    stages {
+        stage('Environment') {
+            steps {
+                sh 'cat /etc/os-release'
+                sh '/opt/rocm/bin/rocm-smi'
+            }
+        }
+        stage('Build libMLIRMIOpen') {
+            steps {
+                checkout scm
+                sh '''
+                    mkdir -p build && cd build
+                    /usr/local/bin/cmake -G "Unix Makefiles" ../llvm \
+                         -DLLVM_ENABLE_PROJECTS="mlir;lld" \
+                         -DCMAKE_BUILD_TYPE=Release \
+                         -DBUILD_SHARED_LIBS=OFF \
+                         -DLLVM_BUILD_LLVM_DYLIB=OFF \
+                         -DLLVM_ENABLE_TERMINFO=OFF
+                    make -j$(nproc) libMLIRMIOpen
+                '''
+            }
+        }
+        stage('Build MIOpenDriver and Test') {
+            steps {
+                checkout scm
+                sh '''
+                    git clone https://github.com/ROCmSoftwarePlatform/MIOpen.git
```

Comment:
Once you go back to home dir, you wouldn't clone in the build folder of `llvm-project-mlir`.

Reply:
I didn't clone in the build folder. I was cloning in the llvm-project-mlir directory. 

I changed the code to clone MIOpen in the /home/jekins/workspace/$BUILD_TAG directory, where $BUILD_TAG is the project name followed by a build id. It is possible, though unlikely, that there exists $BUILD_TAG directory. In this case 'mkdir $BUILD_TAG' would fail and the pipeline would stop.